spec/support/cassettes/Types_User/returns_fields.json

```diff
@@ -0,0 +1 @@
+{"http_interactions":[{"request":{"method":"post","uri":"https://api.github.com/graphql","body":{"encoding":"UTF-8","base64_string":"eyJxdWVyeSI6InF1ZXJ5IEdyYXBoUUxfX0NsaWVudF9fT3BlcmF0aW9uRGVm\naW5pdGlvbl83MDM1NjUyNjAyODY0MCB7XG4gIHAzMG5vZGU6IG5vZGUoaWQ6\nIFwiTURRNlZYTmxjak13T1RJd01qRTJcIikge1xuICAgIF9fdHlwZW5hbWVc\nbiAgICAuLi4gb24gVXNlciB7XG4gICAgICBwNXVybDogdXJsXG4gICAgfVxu\nICAgIC4uLiBvbiBVc2VyIHtcbiAgICAgIHAybG9naW46IGxvZ2luXG4gICAg\nfVxuICAgIC4uLiBvbiBVc2VyIHtcbiAgICAgIHAzYXZhdGFyVXJsOiBhdmF0\nYXJVcmxcbiAgICB9XG4gIH1cbn0iLCJvcGVyYXRpb25OYW1lIjoiR3JhcGhR\nTF9fQ2xpZW50X19PcGVyYXRpb25EZWZpbml0aW9uXzcwMzU2NTI2MDI4NjQw\nIn0=\n"},"headers":{"Accept-Encoding":["gzip;q=1.0,deflate;q=0.6,identity;q=0.3"],"Accept":["application/json"],"User-Agent":["Ruby"],"Content-Type":["application/json"]}},"response":{"status":{"code":200,"message":"OK"},"headers":{"Date":["Wed, 17 Oct 2018 22:07:41 GMT"],"Content-Type":["application/json; charset=utf-8"],"Transfer-Encoding":["chunked"],"Server":["GitHub.com"],"Status":["200 OK"],"X-Ratelimit-Limit":["5000"],"X-Ratelimit-Remaining":["5000"],"X-Ratelimit-Reset":["1539817136"],"Cache-Control":["no-cache"],"X-Oauth-Scopes":["admin:org, admin:org_hook, delete_repo, repo, user:email"],"X-Accepted-Oauth-Scopes":["repo"],"X-Oauth-Client-Id":["\u003cTEST_APPLICATION_GITHUB_CLIENT_ID\u003e"],"X-Github-Media-Type":["github.v4"],"Access-Control-Expose-Headers":["ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval"],"Access-Control-Allow-Origin":["*"],"Strict-Transport-Security":["max-age=31536000; includeSubdomains; preload"],"X-Frame-Options":["deny"],"X-Content-Type-Options":["nosniff"],"X-Xss-Protection":["1; mode=block"],"Referrer-Policy":["origin-when-cross-origin, strict-origin-when-cross-origin"],"Content-Security-Policy":["default-src 'none'"],"Vary":["Accept-Encoding"],"X-Github-Request-Id":["DE2A:4396:A4E92A:CB1855:5BC7B2AD"]},"body":{"encoding":"ASCII-8BIT","base64_string":"eyJkYXRhIjp7InAzMG5vZGUiOnsiX190eXBlbmFtZSI6IlVzZXIiLCJwNXVy\nbCI6Imh0dHBzOi8vZ2l0aHViLmNvbS9kMTIiLCJwMmxvZ2luIjoiZDEyIiwi\ncDNhdmF0YXJVcmwiOiJodHRwczovL2F2YXRhcnMwLmdpdGh1YnVzZXJjb250\nZW50LmNvbS91LzxURVNUX0NMQVNTUk9PTV9PV05FUl9HSVRIVUJfSUQ+P3Y9\nNCJ9fX0=\n"},"http_version":null},"recorded_at":"Wed, 17 Oct 2018 22:07:41 GMT"}],"recorded_with":"VCR 3.0.3"}
```

Comment:
This might be the smallest VCR cassette I've ever seen 😮

Reply:
That's the beauty of GraphQL :D